reveal/theme-zenika/code.css

```diff
@@ -105,4 +105,142 @@ pre .xml .vbscript,
 pre .xml .css,
 pre .xml .cdata {
   opacity: 0.5;
+}*/
+
+/**
+ * prism.js default theme for JavaScript, CSS and HTML
+ * Based on dabblet (http://dabblet.com)
+ * @author Lea Verou
+ */
```

Comment:
Can't this be put in its own file? It would be nicer to separate our own code from vendor code. Also, indentation is not the same as the surrounding file.